ChangeLog.d/add-block-cipher-no-decrypt.txt

```diff
@@ -0,0 +1,6 @@
+Features
+   * Add support to remove decryption operation for cipher type of AES, ARIA
+     and CAMELLIA. A new configuration option, MBEDTLS_BLOCK_CIPHER_NO_DECRYPT
+     can be used to enable this feature.
+     Note that this configuration option is incompatible with MBEDTLS_DES_C,
```

Comment:
Does it need to be incompatible with `DES_C`? It seems to me the ption could be compatible but simply have no effect on DES - or am I missing something?

Reply:
We don't care about builds with DES for this work. If the option has no effect on DES, we have to special-case generic decryption code as still enabled if DES is enabled. If the option has the general effect also on DES, we have to update guards in DES code. Declaring incompatibility is less work and just as much use.

Reply:
Ok, fine with me, let's keep it incompatible then.